patches/minecraft/net/minecraft/client/gui/GuiScreen.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/client/gui/GuiScreen.java
 +++ ../src-work/minecraft/net/minecraft/client/gui/GuiScreen.java
-@@ -147,7 +147,10 @@
+@@ -63,6 +63,7 @@
+     private long field_146288_g;
+     private int field_146298_h;
+     private URI field_175286_t;
++    protected boolean keyHandled, mouseHandled; // Forge: allow canceling key and mouse Post events from handleMouseInput and handleKeyboardInput
```

Comment:
The goal is not to keep the line count the same, it's to minimize the number of added and removed lines.
Shifting the patch numbers of later sections is not adding more lines to the patch.